samples/subsys/usb/console/src/main.c

```diff
@@ -6,29 +6,30 @@
 
 #include <zephyr.h>
 #include <sys/printk.h>
-#include <sys/util.h>
 #include <string.h>
 #include <usb/usb_device.h>
 #include <drivers/uart.h>
 
+#define CDC_ACM_LABEL DT_LABEL(DT_COMPAT_GET_ANY_STATUS_OKAY(zephyr_cdc_acm_uart))
+
 void main(void)
 {
-	const struct device *dev = device_get_binding(
-			CONFIG_UART_CONSOLE_ON_DEV_NAME);
+	const struct device *dev = DEVICE_DT_GET_ONE(zephyr_cdc_acm_uart);
 	uint32_t dtr = 0;
 
 	if (usb_enable(NULL)) {
 		return;
 	}
 
-	/* Poll if the DTR flag was set, optional */
+	/* Poll if the DTR flag was set */
 	while (!dtr) {
 		uart_line_ctrl_get(dev, UART_LINE_CTRL_DTR, &dtr);
+		/* Give CPU resources to low priority threads. */
```

Comment:
Confusing comment here. Is there another thread in the system? I don't see the application starting any; does the USB subsystem start some threads automatically?

Reply:
it does, USB has its own workqueue

Reply:
Thanks, resolving